book/version_control.md

```diff
@@ -371,25 +367,26 @@ To handle a data breach, you should:
    * Discuss the breach with the Data Protection Officer in your department. They should be able to advise you on the steps you should take as well, according to your departments data security policy.
    * If the breach includes pre-publication statistics, you must also report the breach to the [Office for Statistics Regulation](https://osr.statisticsauthority.gov.uk/).
 
-
 ```{warning}
-Deleting a sensitive file and commiting this change is not sufficient. All previous versions of files are retained in a repository's history, so sensitive parts of the file must be entirely removed from the history.
+Deleting a sensitive file and committing this change is not sufficient. All previous versions of files are retained in a repository's history, so sensitive parts of the file must be entirely removed from the history.
 
 The [Pro Git book section on rewriting history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) details methods for editing and deleting files from your repository's commit history. Git has a built-in way to do this. If you are able to install them in your department, you may use one of the simpler alternatives, such as [git-filter-repo](https://github.com/newren/git-filter-repo) (python) or [BFG repo-cleaner tool](https://rtyley.github.io/bfg-repo-cleaner/) (Java).
 ```
 
-## GitHub
+## Using GitHub
 
 A number of version control platforms extend the functionality of Git, to further improve collaborative working.
 
 Here we describe some of the beneficial features supported by [GitHub](https://github.com/), the world's leading software development platform. GitHub provides additional tools for better management of collaborative work. Many of these tools are also discussed in detail on the [GitHub features page](https://github.com/features/project-management/), but we will describe how they may be applied in analytical workflows here.
 
 (issues)=
-### Efficient use of issues
+
+### Use issues efficiently
```

Comment:
Could be "Use issues to...", so that it's actionable

Reply:
Updated to:
### Use Git issues to plan and discuss development of your project